src/client/pages/_slug.vue

```diff
@@ -1,8 +1,14 @@
 <template>
-  <div>
-    <h1>{{ page.title }}</h1>
+  <div class="page" id="page">
+    <header class="page-header">
+        <h1 class="page-header__title">{{ page.title }}</h1>
+        <p class="page-header__subtitle" v-if="page.subtitle">{{ page.subtitle }}</p>
+    </header>
+
     <table-of-contents v-if="page.hasToc" :items="page.tocItems" />
```

Comment:
Wrap this in an element, so you can style it to be on the side while using BEM, instead of needing `.page .table-of-contents`:
```html
<div class="page__sidebar" v-if="page.hasToc">
  <table-of-contents :items="page.tocItems" />
</div>
```